README.md

```diff
@@ -70,14 +71,12 @@ Features that are present but not tested:
 
 ## Fetching
 
-You can fetch/clone this whole repository, but you actually need only
-one file from it: `prod-devel-rcar.yaml`. During build `moulin` will
-fetch this repository again into `yocto/` directory. So, to avoid
-possible confusion, we recommend to download only `prod-devel-rcar.yaml`:
-
-```
-# curl -O https://raw.githubusercontent.com/xen-troops/meta-xt-prod-devel-rcar/master/prod-devel-rcar.yaml
-```
+You can clone this whole repository, or download it as an archive.
+During the build few directories will be created and additional
+dependencies will be fetched into them.
+There will be created the build directory `yocto/` for yocto's
```

Comment:
A bit strange phrasing. Maybe something like this?

"The build system will create build directory `yocto/` for yocto's
meta-layers, and `zephyr/` and/or `android/` directories,
depending on the build options."

Reply:
Fixed.